docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

```diff
@@ -0,0 +1,197 @@
+
+    > **NOTE: the script will automatically uninstall any pre-existing Azure CLI versions in the workstation and will deploy the latest 64 bit version, as it is a requirement to deploy the Resource Bridge**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+- User is editing the onboarding PowerShell script to match the environment (1-time edit).
+
+- User will run the script from the local workstation.
+
+- User will verify the correct onboarding.
+
+## Connect VMware vCenter Server to Azure Arc
+
+- Change the environment variables according to your environment:
+  - _'location'_ - the Azure Region you want to deploy to
+  - _'subscriptionId'_ - your subscription ID
+  - _'resourceGroupName'_ - name of the Azure Resource Group you will create your resources in
+  - _'applianceName'_ - a name for the Bridge appliance
+  - _'customLocationName'_ - a name for the Azure Arc custom location
+  - _'vcenterName'_ - your vCenter name
+  - _'vcenterFqdn'_ - your vCenter fully qualified name
+  - _'vcenterUsername'_ - username to authenticate to vCenter
+  - _'vcenterPassword'_ - password to authenticate to vCenter
+  - _'spnClientId'_ - your service principal App ID
+  - _'spnClientSecret'_ - your service principal password
+  - _'spnTenantId'_ - your Azure AD tenant ID
+  - _'vmTemplate'_ - the Arc appliance template name
+  - _'datacenter'_ - the vSphere data center where the appliance will be deployed
+  - _'datastore'_ - the vSphere data store name where the appliance will be deployed
+  - _'folder'_ - the vSphere folder where the template and appliance will be deployed
+  - _'dnsServer'_ - DNS server to be used for the appliance
+  - _'gateway'_ - Gateway address to be used for the appliance
+  - _'ipAddressPrefix'_ - Network address in CIDR notation to be used by the appliance
+  - _'k8sNodeIpPoolStart'_ - IP range start for the IPs to be used by the appliance
+  - _'k8sNodeIpPoolEnd'_ - IP range end for the IPs to be used by the appliance
+  - _'segment'_ - Name of the virtual network or segment to which the appliance VM must be connected
+  - _'resourcePool'_ - the name of the resource pool to be used by the appliance
+  - _'controlPlaneEndpoint'_ - IP address of the Kubernetes cluster control plane
+  - _'vSphereRP'_ - ConnectedVMwarevSphere resource provider Id 
+  
+  ![Screenshot environment variables](./02.png)
+
+- Once you have provided all of the required environment variables. Open an administrative PowerShell window and run the script with the command:
+
+  ```powershell
+  .\vCenter_onboarding.ps1
+  ```
+
+  ![Screenshot showing script output](./03.png)
+
+  ![Screenshot showing script output](./04.png)
+
+  ![Screenshot showing script output](./05.png)
+
+  ![Screenshot showing script output](./06.png)
+
```

Comment:
In the suggested preview note, we should add something like "as a result, versions captured in the scenario's various screenshots are subject to change". 

Once you will add to this scenario, we will stage another PR to add this across the repo.